Mathlib/RingTheory/Multiplicity.lean

```diff
@@ -481,31 +481,24 @@ theorem finite_mul_aux {p : α} (hp : Prime p) {a b : α} :
     have : p ∣ a * b := ⟨p ^ (n + m) * s, by simp [hs, pow_add, mul_comm, mul_assoc, mul_left_comm]⟩
     (hp.2.2 a b this).elim
       (fun ⟨x, hx⟩ =>
-        have hn0 : 0 < n :=
-          Nat.pos_of_ne_zero fun hn0 => by simp [hx, hn0] at ha
+        have hn0 : 0 < n := Nat.pos_of_ne_zero fun hn0 => by simp [hx, hn0] at ha
         have hpx : ¬p ^ (n - 1 + 1) ∣ x := fun ⟨y, hy⟩ =>
           ha (hx.symm ▸ ⟨y, mul_right_cancel₀ hp.1 <| by
-                  rw [tsub_add_cancel_of_le (succ_le_of_lt hn0)] at hy;
-                    simp [hy, pow_add, mul_comm, mul_assoc, mul_left_comm]⟩)
+            rw [tsub_add_cancel_of_le (succ_le_of_lt hn0)] at hy
+            simp [hy, pow_add, mul_comm, mul_assoc, mul_left_comm]⟩)
         have : 1 ≤ n + m := le_trans hn0 (Nat.le_add_right n m)
-        finite_mul_aux hp hpx hb
-          ⟨s, mul_right_cancel₀ hp.1 (by
-                rw [tsub_add_eq_add_tsub (succ_le_of_lt hn0), tsub_add_cancel_of_le this]
-                simp_all [mul_comm, mul_assoc, mul_left_comm, pow_add])⟩)
+        finite_mul_aux hp hpx hb ⟨s, mul_right_cancel₀ hp.1 (by
+          rw [tsub_add_eq_add_tsub (succ_le_of_lt hn0), tsub_add_cancel_of_le this]
+          simp_all [mul_comm, mul_assoc, mul_left_comm, pow_add])⟩)
       fun ⟨x, hx⟩ =>
-        have hm0 : 0 < m :=
-          Nat.pos_of_ne_zero fun hm0 => by simp [hx, hm0] at hb
+        have hm0 : 0 < m := Nat.pos_of_ne_zero fun hm0 => by simp [hx, hm0] at hb
         have hpx : ¬p ^ (m - 1 + 1) ∣ x := fun ⟨y, hy⟩ =>
-          hb
-            (hx.symm ▸
-              ⟨y,
-                mul_right_cancel₀ hp.1 <| by
-                  rw [tsub_add_cancel_of_le (succ_le_of_lt hm0)] at hy;
-                    simp [hy, pow_add, mul_comm, mul_assoc, mul_left_comm]⟩)
-        finite_mul_aux hp ha hpx
-        ⟨s, mul_right_cancel₀ hp.1 (by
-              rw [add_assoc, tsub_add_cancel_of_le (succ_le_of_lt hm0)]
-              simp_all [mul_comm, mul_assoc, mul_left_comm, pow_add])⟩
+          hb (hx.symm ▸ ⟨y, mul_right_cancel₀ hp.1 <| by
+            rw [tsub_add_cancel_of_le (succ_le_of_lt hm0)] at hy
+            simp [hy, pow_add, mul_comm, mul_assoc, mul_left_comm]⟩)
+        finite_mul_aux hp ha hpx ⟨s, mul_right_cancel₀ hp.1 (by
+          rw [add_assoc, tsub_add_cancel_of_le (succ_le_of_lt hm0)]
+          simp_all [mul_comm, mul_assoc, mul_left_comm, pow_add])⟩
```

Comment:
Idem dito for this kind of reformatting and reflowing of proofs.

Ideally a diff that touches > 30 files will be completely automatically generated by a script that fixes the syntax. Then we can have high confidence that the diff *does one and does it well*.